canonical-websites/snapcraft.io
# Improve the layout of videos on snap details pages

## Done
- Implemented sidebar style screenshot list with video to the left
- No screenshots = video centered
- No video = original carousel
- Mobile = screenshot sidebar drops down to a horizontal, wrapping list below the video
- asciinema has background colour for narrower "videos"

## QA
- Pull the branch
- `./run`
- Visit a publisher listing page
- Add screenshots, no video
- View the public page on both desktop and mobile sizes
- Remove all screenshots
- Add a video
- View the public page on both desktop and mobile sizes
- Add 1 screenshot
- View the public page on both desktop and mobile sizes
- Add additional screenshots and check the public page

- Add and remove screenshots and different sources of video - asciinema, youtube and vimeo

## Screenshots
**Not sure why there are borders around the videos in the screenshots, they're fine on my screen - maybe something to do with the resolution + zoom?**
![screenshot_2019-01-09 install test-lukewh for linux using the snap store snapcraft](https://user-images.githubusercontent.com/479384/50900781-225b0980-140e-11e9-9fa9-3ae3c07cc245.png)
![screenshot_2019-01-09 install test-lukewh for linux using the snap store snapcraft 1](https://user-images.githubusercontent.com/479384/50900790-24bd6380-140e-11e9-9ba8-7ed94bbcfb82.png)
![screenshot_2019-01-09 install test-lukewh-2 for linux using the snap store snapcraft](https://user-images.githubusercontent.com/479384/50900796-27b85400-140e-11e9-85f0-1a9f3760665a.png)
![screenshot_2019-01-09 install test-lukewh-2 for linux using the snap store snapcraft 1](https://user-images.githubusercontent.com/479384/50900800-2a1aae00-140e-11e9-9f6f-166511a368e9.png)
![screenshot_2019-01-09 install lukotron testing for linux using the snap store snapcraft](https://user-images.githubusercontent.com/479384/50900807-2d159e80-140e-11e9-8ae8-c014bbb23ffe.png)
![screenshot_2019-01-09 install lukotron testing for linux using the snap store snapcraft 1](https://user-images.githubusercontent.com/479384/50900812-2e46cb80-140e-11e9-9b7f-aaeb388a788c.png)
